src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+\gammapy side there is not limitation on which kind of analysis can be done
+with which instrument, however in practice it is limited by the availability of
+public data.
+
+\subsection{1D Analysis}
+\label{ssec:1d-analysis}
+One of the most common analysis cases in \gammaray astronomy is measuring the
+spectrum of a source in a given region defined on the sky, in conventional
+astronomy also called aperture photometry. The spectrum is typically measured
+in two steps: first a parametric spectral model is fitted to the data and
+secondly flux points are computed in a pre-defined set of energy bins. The
+result of such an analysis performed on three simulated CTA observations is
+shown in Fig.~\ref{fig:cta_galactic_center}. In this case the spectrum was
+measured in a circular aperture centered on the Galactic Center, in
+\gammaray~astronomy often called "on region". For such analysis the users first
+chooses a region of interest and energy binning, both defined by a
+\code{RegionGeom}. In a second step the events and instrument response are binned
+into maps of this geometry, by the \code{SpectrumDatasetMaker}. All the data and
+reduced instrument response are bundled into a \code{SpectrumDataset}. To estimate
+the expected background in the on region a "reflected regions" background
+method was used~\cite{Berge07}, represented in \gammapy by the
+\code{ReflectedRegionsBackgroundMaker} class. The resulting reflected regions are
+illustrated for all three observations on top of the map of counts. After
+reduction of the data it was modelled using a forward-folding method and a
+power-law spectral shape, using the \code{PowerLawSpectralModel} and \code{Fit} class.
+Based on this best fit model the final flux points and corresponding
+log-likelihood profiles are computed using the \code{FluxPointsEstimator}.
+
+\begin{figure*}[t]
+	\centering
+	\includegraphics[width=1.\textwidth]{figures/cta_galactic_center.pdf}
+	\caption{
+		Example spectral analysis of the Galactic Center for three simulated CTA
+		observations. The left image shows the maps of counts with the measurement
+		region and background regions overlaid in different colors. The right image
+		shows the resulting spectral points and their corresponding log-likelihood
+		profiles.} \label{fig:cta_galactic_center} \end{figure*}
+
+
+\subsection{3D Analysis}
+\label{ssec:3d-analysis}
+The 1D analysis approach is a powerful tool to measure the spectrum of an
+isolated source. However, more complicated situations require a more careful
+treatment. In a field of view containing several overlapping sources, the 1D
+approach cannot disentangle the contribution of each source to the total flux in
+the selected region. Sources with extended or complex morphology can result in
+the measured flux being underestimated, and heavily dependent on the choice of
+extraction region. Additionally, the 1D approach neglects the energy-dependence
+of the PSF.
```

Comment:
This is not true! the PSF is taking into account for the leakage. I propose to remove this sentence.